### PR TITLE
Update for list-based configuration

### PR DESCRIPTION
**Pull request by EdJoJob**

### CODE REVIEW

File: filter_plugins/teku_filters.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            'teku_opts_format': self.tekuOptsFormat,
+        }
+
+    def tekuOptsFormat(self, teku_env_opts: Union[str, List[str]]):
+        if isinstance(teku_env_opts, str):
+            teku_env_opts = teku_env_opts.split()
+
+        escaped_teku_env_opts = [item.replace('"', '\\"') for item in teku_env_opts]
+
+        return f'''"{'" "'.join(escaped_teku_env_opts)}"'''
```
Comment (benjamincburns) [inline filter_plugins/teku_filters.py:20]: lolwut?

Comment (benjamincburns) [reply filter_plugins/teku_filters.py:20]: https://www.python.org/dev/peps/pep-0498/

TIL!

Comment (benjamincburns) [reply filter_plugins/teku_filters.py:20]: This is super not readable, sadly... Perhaps just add a comment like `# quote the individual items`?

**Review by benjamincburns — CHANGES_REQUESTED** [review]

**Comment by joshuafernandes** [comment]
closing this one coz its been superceeded by #7 - if it looks like a list, and quacks like a list...

**Pull request closed without merging**